packages/react-native-codegen/src/parsers/typescript/modules/index.js

```diff
@@ -213,54 +204,28 @@ function translateTypeAnnotation(
             translateTypeAnnotation,
           );
         }
-        case 'Stringish': {
-          return emitStringish(nullable);
-        }
-        case 'Int32': {
-          return emitInt32(nullable);
-        }
-        case 'Double': {
-          return emitDouble(nullable);
-        }
-        case 'Float': {
-          return emitFloat(nullable);
-        }
-        case 'UnsafeObject':
-        case 'Object': {
-          return emitGenericObject(nullable);
-        }
-        case 'Partial': {
-          throwIfPartialWithMoreParameter(typeAnnotation);
-
-          const annotatedElement = parser.extractAnnotatedElement(
-            typeAnnotation,
-            types,
-          );
-
-          throwIfPartialNotAnnotatingTypeParameter(
-            typeAnnotation,
-            types,
-            parser,
-          );
-
-          const properties = parser.computePartialProperties(
-            annotatedElement.typeAnnotation.members,
+        default: {
+          const commonType = emitCommonTypes(
             hasteModuleName,
             types,
+            typeAnnotation,
```

Comment:
Why don't we pass directly: `typeAnnotation.typeName.name`?

Reply:
yeah, you're right we only need the grandchild prop (`name`) of `typeAnnotation`.

Reply:
Actually, the `emitPartial` requires it as param :)